easybuild/easyblocks/i/impi.py

```diff
@@ -210,7 +211,17 @@ def sanity_check_step(self):
             'dirs': [],
         }
 
-        super(EB_impi, self).sanity_check_step(custom_paths=custom_paths)
+        # Add minimal test program to sanity checks
+        impi_testsrc = os.path.join(self.installdir, 'test/test.c')
+        impi_testexe = os.path.join(self.builddir, 'mpi_test')
+        self.log.info("Adding minimal MPI test program to sanity checks: %s", impi_testsrc)
+
+        custom_commands = [
+            "mpiicc %s -o %s" % (impi_testsrc, impi_testexe),  # build test program
+            'mpirun -n %s %s' % (self.cfg['parallel'], impi_testexe),  # run test program
```

Comment:
@lexming We should avoid hardcoding `mpirun` here, see https://github.com/lexming/easybuild-easyblocks/pull/2

Reply:
And mpiicc too since this will also be used by gimpi toolchain

Reply:
Good points! (I was not even aware that `gimpi` was a thing :D)